src/main.js

```diff
@@ -35,8 +35,8 @@ export default function createApp({ apollo = {}, appConfig = {} } = {}) {
 
 	sync(store, router);
 
-	// TODO: Try to get this working without duplicates for server entry.
 	// Checking that sentry is enabled & is not server side
+	// TODO: Enable custom errorHandler for server side to log server errors
 	if (appConfig.enableSentry && typeof window !== 'undefined') {
```

Comment:
@emuvente We could override the enableSentry value to false and simplify this check. It may be also be useful to set a flag for serverEntry as a more generalized value to check though that would still check 2 items.

Reply:
@emuvente merging for now, both server and client entry get executed on the client in some form so forcing this to false in server actually turns it off for client as well.